lumen/sources/intake.py

```diff
@@ -31,11 +31,11 @@ def get_schema(self, table=None):
         for entry in list(self.cat):
             if table is not None and entry != table:
                 continue
-            data = self.get(entry, __dask=True)
-            if self.load_schema:
-                schemas[entry] = get_dataframe_schema(data)['items']['properties']
-            else:
+            elif not self.load_schema:
                 schemas[entry] = {}
+                continue
+            data = self.get(entry, __dask=True)
```

Comment:
Net effect here is to move data pull s.t. it's not performed ` if not self.load_schema` ?

Reply:
Right, creating a dask dataframe is usually cheap but why do it at all when it's not used.